Modix.Services/Moderation/EphemeralUser.cs

```diff
@@ -62,6 +62,13 @@ public EphemeralUser(ulong userId, string name, IGuild guild)
 
         public string VoiceSessionId { get; private set; } = null;
 
+        public IActivity Activity => throw new NotImplementedException();
+
+        //Discord default green user icon
+        public string GetDefaultAvatarUrl() => "https://discordapp.com/assets/dd4dbc0016779df1378e7812eabaa04d.png";
```

Comment:
Could we host a copy of this ourselves?

Reply:
Actually thinking about it, we literally wouldn't use this anywhere since it only exists as glue from the  API to the database, and we don't store user avatars. I'll just remove it.